### PR TITLE
add splash page for quarkus kafka project

### PR DESCRIPTION
**Pull request by josiemundi**
### Checklist:

- [x] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [x] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [x] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure).

### Modifying an existing stack:

- [x] Updated the stack version in `stack.yaml` 

I have added a splash page for the root of the starter application in the Quarkus stack for Kafka template. Previously, there was not one and this was causing issues as outlined in the one at the following link:

### CODE REVIEW

**Comment by neeraj-laad** [comment]
@josiemundi Thanks for your contribution! Please can you review and sign our CLA before we look at this PR.

File: incubator/quarkus/templates/kafka/src/main/resources/META-INF/resources/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                <li><a href="https://quarkus.io/guides/getting-started.html">Getting started</a></li>
+                <li><a href="https://quarkus.io">Quarkus Web Site</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+
+</body>
+</html>
```
Comment (paulrobinson) [inline incubator/quarkus/templates/kafka/src/main/resources/META-INF/resources/index.html:152]: Can you add the new-line to the end of the file? We use diff to make sure the code templates stay in sync with the Quarkus quickstarts they are based on. This flags the file as having differences, so will make maintenance harder when we update the Quarkus version and the template code. 

I fixed this on my Mac with:

```
sed -i '' -e '$a\' incubator/quarkus/templates/kafka/src/main/resources/META-INF/resources/index.html
```

I'm not smart enough to know why adding a new line in vi didn't work for me!

**Comment by paulrobinson** [comment]
@ianpartridge do you want to review this too? I see you have an interest in the index.html file and discuss it in https://github.com/quarkusio/quarkus-quickstarts/issues/617

**Comment by ianpartridge** [comment]
Thanks Paul. I'm happy with this - our need was for this template to have something at the root so when a user deploys it on OCP and clicks "Open URL" they don't see an error.  Adding this static page absolutely fulfils that (and moves this closer in sync with the upstream quickstart too 👍 )

File: incubator/quarkus/templates/kafka/src/main/resources/META-INF/resources/index.html (hunk 1 above)
Comment (josiemundi) [reply incubator/quarkus/templates/kafka/src/main/resources/META-INF/resources/index.html]: I have added this now :) thanks!

**Pull request merged**